src/components/Questions/Header.vue

```diff
@@ -7,6 +7,14 @@
       @click="togglePalette"
       data-test="togglePaletteButton"
     ></icon-button>
+    <div class="flex space-x-3">
+      <!-- timer button / can't click -->
+      <icon-button
+      v-if="!hasQuizEnded && $props.hasTimeLimit"
```

Comment:
```suggestion
      v-if="!hasQuizEnded && hasTimeLimit"
```
Just this should be enough no?